### DISCUSSION TITLE
Looking for a sample showing how to modify a polygon dragging points

### DISCUSSION THREAD

**Discussion by upswing1** [Mapsui/Mapsui#2069] [Q&A]
I would like to learn how to draw then select a polygon and modify its shape by dragging some points interactively. Can anyone please share some code or create a sample that the community could use.

**Comment by pauldendulk** [comment]
If you go to https://mapsui.com/samples/ and select category editing there are various samples. The source code on the webpage is not very helpful so you need to clone the Mapsui repository to see it.

**Reply by upswing1** [reply]
Thank you

**Comment by upswing1** [comment]
Nuget packet manager for latest rc3 may have a problem installing the latest rc version. After installing the latest version it shows classes missing properties etc.  However reading the source code shows the properties are implemented.

**Comment by inforithmics** [comment]
Editing is currently only implemented in Wpf and Blazor the next version has more platforms implementing editing

**Reply by upswing1** [reply]
The next version meaning rc4?

**Reply by pauldendulk** [reply]
4.0.0. Just released.

**Reply by upswing1** [reply]
great news
